src/profiles_management/helpers/profiles.py

```diff
@@ -4,9 +4,16 @@
 from typing import Iterator
 
 from lightkube import Client
-from lightkube.generic_resource import GenericGlobalResource, create_global_resource
+from lightkube.generic_resource import (
+    GenericGlobalResource,
+    GenericNamespacedResource,
+    create_global_resource,
+)
+from lightkube.types import PatchType
 
 from profiles_management.helpers import k8s
+from profiles_management.helpers.k8s import ensure_namespace_exists
+from profiles_management.pmr import classes
 
 Profile = create_global_resource(
```

Comment:
suggestion: I got confused while reading the function, we should probably rename this to something more descriptive:
```suggestion
Profile_lightkube_resource = create_global_resource(
```
or something similar

Reply:
I see what you mean, what about `ProfilesLightkube`?

Doing a pass to see how it looks like, but should indeed improve the readability

Reply:
WDYT?
https://github.com/canonical/github-profiles-automator/pull/27/commits/e88e601e6469215d90543a11d67395b9e28c13c1

Reply:
Not 100% sure if PascalCase is recommended for this, but seems good!